src/Components/Navbar/nav.jsx

```diff
@@ -1,17 +1,25 @@
-import React from "react";
+import React from 'react';
 
-const NavBar = () => {
-  return (
-    <nav className="nav">
-      <div>
-        <a>Profile</a>
-      </div>
-      <div>Message</div>
-      <div>News</div>
-      <div>Music</div>
-      <div>Settings</div>
-    </nav>
-  );
-};
+import style from './nav.module.css';
+
+const NavBar = () => (
+  <nav className={style.nav}>
+    <div className={style.item}>
+      <a href="/profile">Profile</a>
+    </div>
+    <div className={`${style.item} ${style.active}`}>
+      <a href="/dialogs">Message</a>
+    </div>
+    <div className={style.item}>
+      <a href="/news">News</a>
+    </div>
+    <div className={style.item}>
+      <a href="/music">Music</a>
```

Comment:
Consider storing these href paths at the core level in routes.js

Reply:
I've created  route.js in "src" folder. And i've used this object in App.js and nav.js